Gemfile

```diff
@@ -1,7 +1,8 @@
 source "https://rubygems.org"
 
 gem "rails", ">= 4.0.0"
-gem "kaminari"
+# FIXME: Fix garage to work with kaminari v1.
+gem "kaminari", "< 1"
```

Comment:
This is used by not garage but dummy rails app, isn't it? So I think it's reasonable.